src/main/webapp/course-style.css

```diff
@@ -0,0 +1,78 @@
+.flex-container {
+    display: flex;
+    height: 600px;
+}
+
+.flex-child {
+    flex: 1;
+}  
+
+.form-container {
+    padding-top: 100px;
+}
+
+.my-3 {
+    margin: 20px 125px;
+}
+
+input[type="number"],
+textarea {
+  border: 1px solid #228b22;
+  border-radius: 4px;
+  box-sizing: border-box;
+  display: block;
+  font-size: large;
+  margin: 8px 0;
+  padding: 12px 20px;
+  text-align: center;
+  width: 70%;
+}
+
+input[type="submit"] {
+  background-color: #4caf50;
+  border: none;
+  border-radius: 4px;
+  color: white;
```

Comment:
FYI colors like this ("white", "black", etc) often are different across browsers. So in prod code we always specify in hex instead.